linux/kernel/building.md

```diff
@@ -93,23 +94,23 @@ source ~/.bashrc
 ```
 ### Get sources
 
-To download the minimal source tree for the current branch:
+To download the minimal source tree for the current branch, run:
 
 ```bash
 git clone --depth=1 https://github.com/raspberrypi/linux
 ```
 
-See [**Choosing sources**](#choosing_sources) above for instructions of how to choose a different branch.
+See [**Choosing sources**](#choosing_sources) above for instructions on how to choose a different branch.
 
 ### Build sources
 
-To download the kernel source code, see "Get sources" above.
+To download the kernel source code, see [**Get sources**](#get_sources) above.
```

Comment:
I think this line is redundant now.

Reply:
I agree, I think anyone reading this would be perfectly aware on how to download the source given the preceding paragraphs. Feel free to update, since the rest o the change has been through CE, I'll jsut merge it.